app/js/components/homePage/manageApps/deleteAddonModal.jsx

```diff
@@ -8,33 +8,32 @@
  * graphic logo is a trademark of OpenMRS Inc.
```

Comment:
Could we have tests for this particular file? Or can we handle testing this  in a separate ticket?

Reply:
It's handled in the next ticket